app/src/main/java/com/nononsenseapps/feeder/openai/OpenAIClient.kt

```diff
@@ -40,6 +42,7 @@ class OpenAIClientDefault(
 private fun OpenAISettings.toOpenAIConfig(): OpenAIConfig =
     OpenAIConfig(
         token = key,
+        timeout = Timeout(socket = timeoutSeconds.seconds),
```

Comment:
```suggestion
        timeout = Timeout(socket = timeoutSeconds.seconds).coerceIn(30, 600),
```

I think this is the best place to put the coercion since it's where it is actually used

Reply:
I think this is where the openai settings for feeder is converted to OpenAI client library config? So I thought putting the cocercion here will not save the correct value after cocercion into settingsStore, that's why I put the cocercion to settingsStore. I can try again later today.

Reply:
This will not "save" the coercion but that is not the point.

The point is to ensure a reasonable value is used for the connection even if there is "garbage" in the settings file

Reply:
You validation rules will ensure values are in the proper range but bugs (now and especially in the future) are possible